apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/SpannerStressTests.cs

```diff
@@ -131,6 +132,10 @@ public SpannerStressTests(TestDatabaseFixture testFixture, ITestOutputHelper out
 
         private async Task RunStress(Func<Stopwatch, Task<TimeSpan>> writeFunc)
         {
+            // Clear current session pool to eliminate the chance of a previous
```

Comment:
Is this logically part of this change, or just an issue you noticed while testing this change?